lib/ansible/modules/service.py

```diff
@@ -1010,8 +1016,21 @@ def get_service_tools(self):
 
         self.sysrc_cmd = self.module.get_bin_path('sysrc')
 
+    def run_service_cmd(self, status=None):
+        if not status:
+            return (None, '', '')
+
+        cmd = f"{self.svc_cmd} "
+        # Jail name takes precedence over other arguments
+        if self.arguments.startswith(('-j', '-v')):
+            cmd += f"{self.arguments} {self.name} {status}"
+        else:
+            cmd += f"{self.name} {status} {self.arguments}"
+        return self.execute_command(cmd)
```

Comment:
I think this would be more readable:

```suggestion
        # Jail name takes precedence over other arguments
        if self.arguments.startswith(('-j', '-v')):
            cmd = f"{self.svc_cmd} {self.arguments} {self.name} {status}"
        else:
            cmd = f"{self.svc_cmd} {self.name} {status} {self.arguments}"

        return self.execute_command(cmd)
```